tools/syntax_updater/from_0_1_0.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright © SixtyFPS GmbH <info@sixtyfps.io>
+// SPDX-License-Identifier: (GPL-3.0-only OR LicenseRef-SixtyFPS-commercial)
+
+use std::io::Write;
+
+use sixtyfps_compilerlib::parser::{syntax_nodes, SyntaxNode};
+
+/// Rename `loop-count` to `iteration-count` in `PropertyAnimation`s
+pub(crate) fn fold_node(
+    node: &SyntaxNode,
+    file: &mut impl Write,
+    state: &mut crate::State,
+) -> std::io::Result<bool> {
+    if let Some(binding) = syntax_nodes::Binding::new(node.clone()) {
+        if state.property_name.as_deref().unwrap_or_default() == "loop-count"
+            && has_parent_anim(&binding.clone().into())
```

Comment:
I think you also need to consider loop_count with underscore

Reply:
I expected earlier steps to have normalized to `loop-count` already. Is that not the case?

Reply:
nope, rowan is not really an AST, more like a concrete syntax tree. It just has the raw data.

Reply:
You are right: I somehow missed the code in the `fold_token` function in `main.rs` is just an example:-)